src/rf/apps/core/models.py

```diff
@@ -177,11 +187,49 @@ def to_json(self):
             'tile_origin': self.tile_origin,
             'resampling': self.resampling,
             'transparency': self.transparency,
-            'status': self.status,
+            'status_created': (True if self.status_created is not None
+                               else False),
+            'status_upload_start': (True if self.status_upload_start
+                                    is not None else False),
+            'status_upload_end': (True if self.status_upload_end is not None
+                                  else False),
+            'status_validate_start': (True if self.status_validate_start
+                                      is not None else False),
+            'status_validate_end': (True if self.status_validate_end
+                                    is not None else False),
+            'status_thumbnail_start': (True if self.status_thumbnail_start
+                                       is not None else False),
+            'status_thumbnail_end': (True if self.status_thumbnail_end
+                                     is not None else False),
+            'status_create_cluster_start': (True if
+                                            self.status_create_cluster_start
+                                            is not None else False),
+            'status_create_cluster_end': (True if
+                                          self.status_create_cluster_end
+                                          is not None else False),
+            'status_chunk_start': (True if self.status_chunk_start is not None
+                                   else False),
+            'status_chunk_end': (True if self.status_chunk_end is not None
+                                 else False),
+            'status_mosaic_start': (True if self.status_mosaic_start
+                                    is not None else False),
+            'status_mosaic_end': (True if self.status_mosaic_end is not None
+                                  else False),
+            'status_failed': (True if self.status_failed is not None
+                              else False),
+            'status_completed': (True if self.status_completed is not None
+                                 else False),
+            'status_upload_error': self.status_upload_error,
+            'status_validate_error': self.status_validate_error,
+            'status_thumbnail_error': self.status_thumbnail_error,
+            'status_create_cluster_error': self.status_create_cluster_error,
+            'status_chunk_error': self.status_chunk_error,
+            'status_mosaic_error': self.status_mosaic_error,
+            'status_failed_error': self.status_failed_error,
```

Comment:
Since each job has a separate error field, it doesn't seem like `status_failed_error` is needed.